tests/periph_flashpage/main.c

```diff
@@ -363,6 +370,35 @@ static int cmd_test_last(int argc, char **argv)
     puts("wrote local page buffer to last flash page");
     return 0;
 }
+
+/**
+ * @brief   Does a write and verify test on reserved page
+ */
+static int cmd_test_res(int argc, char **argv)
+{
+    (void) argc;
+    (void) argv;
+
+    char fill = 'a';
+    unsigned page = flashpage_page((void *)_backing_memory);
+
+    printf("Reserved page num: %u \n", page);
+
+    for (unsigned i = 0; i < sizeof(page_mem); i++) {
+        page_mem[i] = (uint8_t)fill++;
+        if (fill > 'z') {
+            fill = 'a';
+        }
+    }
+
+    if (flashpage_write_and_verify(page, page_mem) != FLASHPAGE_OK) {
+        puts("error verifying the content of reserved page");
+        return 1;
+    }
+
+    puts("wrote local page buffer to reserved flash page");
```

Comment:
```suggestion
    puts("wrote local page buffer to reserved flash page");
    puts("\nWhen running on a bootloader, as an extra check, try restarting the board and check "
         "whether this application still comes up.");
```

Reply:
In combination with testing static initialization, it might be a nice touch to statically initialize the memory to, say, 0x00 0x42 0xff, check for whether the 0x42 is really there, print "Since the last firmware update this test has been run flash[0] time(s)", and increment these bytes during the write-verify run. Doesn't change anything about the critical step of checking-after-reboot (as a misconfigured checking bootloader would not come up either way), but when it *does* come up it gives the user better reassurance that things did indeed go well.

Reply:
Since the section is not part of the firmware I can not statically initialize the memory. 
I added the print statement as you suggested.